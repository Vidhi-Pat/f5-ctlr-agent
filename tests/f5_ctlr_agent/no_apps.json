{
    "resources": {
        "k8s": {
            "iRules": [],
            "iapps": [],
            "internalDataGroups": [],
            "l7Policies": [],
            "monitors": [],
            "pools": [],
            "virtualServers": []
        }
    }
}
