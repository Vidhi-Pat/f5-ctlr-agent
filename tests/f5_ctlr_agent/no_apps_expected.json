{
    "ltm": {
        "iRules": [],
        "iapps": [],
        "internalDataGroups": [],
        "l7Policies": [],
        "monitors": [],
        "pools": [],
        "virtualServers": []
    },
    "network": {}
}
